src/components/AssetManagementPanel/AssetManagementForm/AssetManagementDropdown/AssetManagementDropdown.tsx

```diff
@@ -1,30 +1,36 @@
-import React from "react";
 import styled from "@emotion/styled";
-import { AssetManagementActions } from "./../../AssetManagementActions";
+import React from "react";
 import { Dropdown } from "react-bootstrap";
 import { mixin, vars } from "../../../../styles";
+import { AssetManagementActions } from "./../../AssetManagementActions";
 
 interface ManageAssetsDropdownProps {
   onSetFormAction: (nextFormAction: AssetManagementActions) => void;
   className?: string;
   canSurrender: boolean;
+  isHolder: boolean;
```

Comment:
```suggestion
  canChangeHolder: boolean;
```

to follow the convention above. At this level of the app the app shouldn't really care what criteria is it to do transfer

Reply:
ok understand.